### ISSUE TITLE
display property on css

### ISSUE THREAD

**Issue by AlaaA0813** [Chedler1/more-css-practice#2]
you set the default display on your html page to block.  i don't think you need to set the display to any other element as block since thats already the default.

**Comment by zachfedor** [comment]
You're right, @AlaaA0813, but which page are you talking about? This issue would be more helpful with the link to the file and line number you're referring to:

https://github.com/Chedler1/more-css-practice/blob/a67f48f82b1c94cbb59bd6f415a6a9b75ae84e3b/instagram/styles.css#L1-L4

Also, the colon after an element selector is invalid syntax and will break the selector!
